script/wrapped-message-tests.js

```diff
@@ -0,0 +1,159 @@
+"use strict";
+
+var assert, config, errCount, Plugin, runNext, testCounter, tests;
+
+assert = require("assert");
+config = require("./manual-testing-config");
+errCount = 0;
+Plugin = require("../../../");
+testCounter = 0;
+tests = [
+    {
+        name: "heartbeat",
+        query: "INSERT INTO ?? (hash, isProcessing, owner, messageBase64, message, topic) VALUES('123', true, ?, '456', 'message', 'Test Topic');"
```

Comment:
This is extremely MySQL specific.  I don't see how it would be possible for the MongoDB-based plugin to benefit from this in any way.